Docs/user_guide/data_loading_and_saving.md

```diff
@@ -58,6 +58,18 @@ Multiple nodes can be exported at once by placing them into a folder and then by
 
 ## Supported Data Formats
 
+:::{note}
+
+### On use of LPS/RAS coordinate systems
+
+DICOM and most medical imaging software uses the **LPS coordinate system** for storing all data. Axis directions of this coordinate system are patient left, posterior, and superior. Unit is millimeters. The choice of origin is arbitrary because only relative differences have meaning, so there is no universal standard, but it is often some geometric center of the imaging system is chosen to be near the center of some object of interest.
```

Comment:
```suggestion
DICOM and most medical imaging software uses the **LPS coordinate system** for storing all data. Axis directions of this coordinate system are patient left, posterior, and superior. Unit is millimeters. The choice of origin is arbitrary because only relative differences have meaning, so there is no universal standard, but it is often set to some geometric center of the imaging system, or it is chosen to be near the center of an object of interest.
```